group_vars/all/dcos.yaml.example

```diff
@@ -2,7 +2,7 @@
 dcos:
   download: "https://downloads.dcos.io/dcos/stable/1.13.4/dcos_generate_config.sh"
   download_checksum: "sha256:55b096236a6de1a40259e6d8a8a21861c5333cbe2b9ba3e274d5de1534b095dc"
-  download_win: "https://downloads.dcos.io/dcos/testing/master/windows/dcos_generate_config_win.sh"
+  download_win: "https://downloads.dcos.io/dcos/stable/master/windows/dcos_generate_config_win.sh"
```

Comment:
this is still not pointing to a release. Looking at the file I wonder if this would actually work having a 1.13.4 and a master based cluster.

Such examples need to be perfect especially if we add such pitfalls as a second download URL